stdlib/stdlib.ml

```diff
@@ -83,23 +83,9 @@ external ( != ) : ('a[@local_opt]) -> ('a[@local_opt]) -> bool = "%noteq"
 
 (* Boolean operations *)
 
-<<<<<<< HEAD
-external not : (bool[@local_opt]) -> bool = "%boolnot"
-external ( & ) : (bool[@local_opt]) -> (bool[@local_opt]) -> bool = "%sequand"
 external ( && ) : (bool[@local_opt]) -> (bool[@local_opt]) -> bool = "%sequand"
 external ( or ) : (bool[@local_opt]) -> (bool[@local_opt]) -> bool = "%sequor"
```

Comment:
I think you removed `not` here where you wanted to remove `( or )` instead.

Reply:
Indeed, thanks for catching that.